docs/develop/pivottables.md

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Work with PivotTables in Office Scripts'
+description: 'The object model for PivotTables in the Office Scripts JavaScript API'
```

Comment:
```suggestion
description: 'Learn about the object model for PivotTables in the Office Scripts JavaScript API.'
```
Description metadata should include a CTA, if possible. Also, since it's included in SERP, end with a period.